terraform/shared/shared.tf.json.template.py

```diff
@@ -551,7 +551,7 @@ def paren(s: str) -> str:
                     'alarm_name': config.qualified_resource_name(a.name, suffix='.alarm'),
                     'comparison_operator': 'GreaterThanOrEqualToThreshold',
                     'evaluation_periods': 1,
-                    'metric_name': a.metric_name,
+                    'metric_name': f'${{aws_cloudwatch_log_metric_filter.{a.name}.metric_transformation[0].name}}',
```

Comment:
We prefer using % for string interpolation in conjunction with Terraform's {} syntax,

```suggestion
                    'metric_name': '${aws_cloudwatch_log_metric_filter.%s.metric_transformation[0].name}' % a.name,
```

Reply:
My suggestion did not account for the line length (my bad), the line is too long.